test/dockerfiles/Dockerfile.universal

```diff
@@ -43,6 +44,7 @@ ADD /build/artifacts-linux-$ARCH/kumactl/kumactl /usr/bin
 ADD /build/artifacts-linux-$ARCH/test-server/test-server /usr/bin
 ADD /test/server/certs/server.crt /kuma
 ADD /test/server/certs/server.key /kuma
+COPY --from=grpcurl /bin/grpcurl /usr/bin/grpcurl
```

Comment:
There's a whole grpc client/server in the `test/server` can we just use and expand this instead of adding a bunch of external stuff?

Reply:
With the current client/server there is no way to inspect the response, you'd have to grep logs so to me it's an improvement. I wouldn't say it's a "bunch of external stuff" it's just one additional binary. Side note: I've handed over this investigation to @Automaat so if he finds a better way to do this then it's fine by me.